js/apps/account-ui/src/applications/Applications.tsx

```diff
@@ -154,8 +155,9 @@ const Applications = () => {
             </DataListItemRow>
 
             <DataListContent
+              id={`content-${application.clientId}`}
               className="pf-u-pl-4xl"
-              aria-label={t("applicationDetails")}
+              aria-label={`${t("applicationDetails")}-${application.clientId}`}
```

Comment:
An ARIA label should be human readable, if you want to interpolate a string value into a translated text you will have to do so in an expression of the label itself:

```ts
t("applicationDetails", { clientId: application.clientId })
```

Reply:
I see, thank you @jonkoops !